backend/actions/Model/updateDocuments.js

```diff
@@ -41,6 +41,23 @@ module.exports = ({ db }) => async function updateDocuments(params) {
       } else if (value === 'undefined') {
         // Use $unset to remove the field for undefined values
         unsetFields[key] = 1;
+      } else if (value === '') {
+        // Handle empty strings - check if field is optional ObjectId or optional field
+        const schemaPath = Model.schema.paths[key];
+        if (schemaPath) {
+          const isObjectId = schemaPath.instance === 'ObjectId';
+          const isRequired = schemaPath.options?.required === true;
+          // If it's an ObjectId field or an optional field, treat empty string as undefined
+          if (isObjectId || !isRequired) {
```

Comment:
Why is this an "or"? Maybe we should just treat `''` as unsetting for all fields regardless of whether they are objectid or required?